introduction.md

```diff
@@ -0,0 +1 @@
+My name is Pablo and I'm here to learn the secret techniques of the octopus cat kung fu
```

Comment:
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.